core/domain/exp_services.py

```diff
@@ -823,30 +823,31 @@ def update_exploration(
     if change_list is None:
         change_list = []
 
-    if is_by_voice_artist and not is_voiceover_change_list(change_list):
-        raise utils.ValidationError(
-            'Voice artist does not have permission to make some '
-            'changes in the change list.')
-
-    is_public = rights_manager.is_exploration_public(exploration_id)
-    if is_public and not commit_message:
-        raise ValueError(
-            'Exploration is public so expected a commit message but '
-            'received none.')
-
-    if (is_suggestion and (
-            not commit_message or
-            not commit_message.startswith(
-                feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX))):
-        raise ValueError('Invalid commit message for suggestion.')
-    if (not is_suggestion and commit_message and commit_message.startswith(
-            feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX)):
-        raise ValueError(
-            'Commit messages for non-suggestions may not start with \'%s\'' %
-            feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX)
+    if not for_testing:
```

Comment:
I think we should not be putting this condition here, since many tests check for validation errors that are raised. Hence, the backend_tests are failing.